ddtrace/tracer/option.go

```diff
@@ -119,6 +119,12 @@ var (
 	defaultRateLimit = 100.0
 )
 
+// Supported trace protocols.
+const (
+	traceProtocolV04 = 0.4 // v0.4 (default)
+	traceProtocolV10 = 1.0 // v1.0
```

Comment:
Super small nit, but my slight concern is that this looks like we're saying `v10` instead of `v1.0`, lol. In the situation that we eventually develop a `v10` (???), maybe we can call this `traceProtocolV1`?

Reply:
I see your point. I changed it to `traceProtocolV1`.